app/components/mandatenbeheer/bestuursorgaan-mandaten.hbs

```diff
@@ -1,4 +1,7 @@
-<div class="au-o-box no-pagination" {{did-insert this.initialize.perform}}>
+<div
+  class="au-o-box no-pagination"
+  {{did-update this.initialize.perform @mandaten}}
```

Comment:
```suggestion
  {{did-update this.initialize.perform @mandaten}}
  {{did-insert this.initialize.perform}}
```

Reply:
you need both or on first render the orderedMandaten is not computed

Reply:
like we said, it would be much more clean to do this in the route but i think you created a story for that?